qiskit/primitives/utils.py

```diff
@@ -25,8 +25,13 @@
 from qiskit.quantum_info import PauliList, SparsePauliOp, Statevector
 from qiskit.quantum_info.operators.base_operator import BaseOperator
 from qiskit.quantum_info.operators.symplectic.base_pauli import BasePauli
+from qiskit.utils.deprecation import deprecate_func
 
 
+@deprecate_func(
+    since="1.2",
+    additional_msg="Use ``QuantumCircuit.initialize`` instead.",
```

Comment:
This is a totally minor suggestion, but this function only acted if the input was a `Statevector` instance, so we could maybe be a bit more precise in the message:

```suggestion
    additional_msg="To initialize a circuit from a ``Statevector`` instance, use ``QuantumCircuit.initialize`` instead.",
```

Reply:
This suggestion might require running `black` or fixing "line too long" lint complaints.

Reply:
Thanks. I will apply your suggestion and apply black.

Reply:
done